alchemy/src/planetscale/branch.ts

```diff
@@ -164,6 +165,20 @@ export const Branch = Resource(
       : typeof props.parentBranch === "string"
         ? props.parentBranch
         : props.parentBranch.name;
+    const organization =
+      props.organization ??
+      (typeof props.database !== "string"
+        ? props.database.organization
+        : (process.env.PLANETSCALE_ORGANIZATION ??
+          process.env.PLANETSCALE_ORG_ID));
+    const database =
+      typeof props.database === "string" ? props.database : props.database.name;
+
+    if (!organization) {
+      throw new Error(
+        "PlanetScale organization is required. Please set the `organization` property or the `PLANETSCALE_ORGANIZATION` environment variable.",
+      );
+    }
```

Comment:
Just checking - but if someone has existing code on the old version, will it:
1. error if they run without checking types? 
2. will deletes and updates work or do we need to migrate state somehow?